ui/src/server/handlers/api.js

```diff
@@ -3012,6 +3012,33 @@ Fetchr.registerService({
     },
 });
 
+Fetchr.registerService({
+    name: 'resource-access',
+    read(req, resource, params, config, callback) {
+        req.clients.zms.getResourceAccessList(
+            { action: params.action, principal: 'user.' + req.session.shortId },
```

Comment:
Instead of using `user.`, can you use appConfig.userDomain
Similar to https://github.com/AthenZ/athenz/blob/004da941e7356f791bbd9cefde3f636357440359/ui/src/server/handlers/api.js#L461

Reply:
fixed

Reply:
@jimmytsang this one isn't fixed yet

Reply:
Sorry about that. Weird, I must've fixed it in another branch or something.

Reply:
fixed